packages/react-native-sdk/docusaurus/docs/reactnative/04-ui-components/call/call-top-view.mdx

```diff
@@ -0,0 +1,80 @@
+---
+id: call-top-view
+title: CallTopView
+---
+
+The `CallTopView` represents the header component that gives the user more information when in a call, while adding a few actions they can trigger while the call is active. The header is useful for showing the call name or title, as well as the state, such as if the user is fully connected to the call or not.
+It has a back button by default and a participant info badge.
+
+![Preview of the CallTopView component](../../assets/04-ui-components/call/call-top-view/call-top-view.png)
+
+## General Usage
+
+To use the component, you can simple use it as:
+
+```tsx
+import {
+  Call,
+  CallTopView,
+  StreamCall,
+} from '@stream-io/video-react-native-sdk';
+
+const VideoCallUI = () => {
+  let call: Call;
+
+  return (
+    <StreamCall call={call}>
+      // highlight-next-line
```

Comment:
You can use the metadata for highlighting - https://docusaurus.io/docs/next/markdown-features/code-blocks#highlighting-with-metadata-string